config/routes.rb

```diff
@@ -5,22 +5,30 @@
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'home#index'
+  root 'tasks#index'
 
   # Example of regular route:
   #   get 'products/:id' => 'catalog#view'
-  get 'tasks/new' => 'home#new'
-  post 'tasks' => 'home#create'
+  get 'tasks/new' => 'tasks#new'
+  post 'tasks' => 'tasks#create'
+  patch 'tasks' => 'tasks#mark_done'
 
-  get 'tasks/:id' => 'home#show'
-  
-  get '/delete_task/:id' => 'home#confirm_remove'
-  delete '/delete_task/:id' => 'home#delete'
+  get 'tasks/:id' => 'tasks#show'
+  get 'tasks/:id/edit' => 'tasks#edit'
+  patch 'edit_task/:id' => 'tasks#update'
+
+  get '/delete_task/:id' => 'tasks#confirm_remove'
```

Comment:
This surprised me and I was impressed you built out the view for confirmation. It's nice and on-brand. 

There's a common practice to use a confirmation dialog in the browser to confirm deletion (see ["confirm: 'question?'" option on `link_to`](http://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to)), which you can do on the link and not have to build an extra form/route/action. But this is pretty cool.